server/src/main/java/org/opensearch/tasks/TaskInfo.java

```diff
@@ -281,6 +328,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             resourceStats.toXContent(builder, params);
             builder.endObject();
         }
+        if (cancelledAt != null) {
+            builder.humanReadableField("cancelled_at_millis", "cancelled_at", new TimeValue(cancelledAt));
```

Comment:
Prefer "cancellation_time_millis"

Reply:
In that does ```cancellation_start_time_millis``` sounds better?
Tagging @dblock as well in case he thinks otherwise.